src/patternfly/components/Hint/hint.scss

```
@@ -1,7 +1,7 @@
 @use '../../sass-utilities' as *;
 
 @include pf-root($hint) {
-  --#{$hint}--GridRowGap: var(--pf-t--global--spacer--sm);
+  --#{$hint}--GridRowGap: var(--pf-t--global--spacer--gap--group--vertical);
```

Per @lboehling this is a good candidate for a more content specific token, so we can use a generic t-shirt spacer for now.

```suggestion
  --#{$hint}--GridRowGap: var(--pf-t--global--spacer--sm);
```